packages/launchpad/src/setup/InstallDependencies.vue

```diff
@@ -6,20 +6,12 @@
     :next-fn="confirmInstalled"
     class="max-w-640px relative"
     :main-button-variant="canNavigateForward ? 'primary' : 'pending'"
+    :skip-fn="!canNavigateForward ? confirmInstalled : undefined"
```

Comment:
Should we always show this? Is there ever a case where this will not be shown?

Reply:
Currently it's not being shown when the user can continue - otherwise it always shows the skip, bc, can be confusing clicking skip and actually doing what continue does